pkg/core/blockchain.go

```diff
@@ -2467,6 +2467,9 @@ func (bc *Blockchain) verifyAndPoolTx(t *transaction.Transaction, pool *mempool.
 		// Only one %w can be used.
 		return fmt.Errorf("%w: %v", ErrPolicy, err)
 	}
+	if t.SystemFee > bc.config.MaxBlockSystemFee {
```

Comment:
But how about `ValidUntilBlock` value? It will be removed from the pool eventually, isn't it?

Reply:
It will, but it will take some time.